bollettino-d-archeologia-online.csl

```diff
@@ -0,0 +1,237 @@
+          <text term="ibid" text-case="capitalize-first" font-style="italic"/>
+        </else-if>
+        <else>
+          <group>
+            <text macro="author-short" suffix=" "/>
+            <text macro="year-date"/>
+            <text macro="locator"/>
+          </group>
+        </else>
+      </choose>
+    </layout>
+  </citation>
+  <bibliography>
+    <sort>
+      <key macro="author"/>
+      <key variable="issued"/>
+      <key variable="page-first"/>
+      <key variable="title"/>
+    </sort>
+    <layout suffix=".">
+      <group>
+        <choose>
+          <if variable="author">
+            <names variable="author">
+              <name form="short" font-variant="small-caps" delimiter=", " et-al-min="3" et-al-use-first="1"/>
+              <et-al font-variant="normal" font-style="italic"/>
+            </names>
+          </if>
+          <else-if variable="editor">
+            <names variable="editor">
+              <name form="short" font-variant="small-caps" delimiter=", " et-al-min="3" et-al-use-first="1"/>
+              <et-al font-variant="normal" font-style="italic"/>
+            </names>
+          </else-if>
+          <else>
+            <text variable="title-short" font-variant="small-caps"/>
+          </else>
+        </choose>
+        <text macro="year-date" prefix=" " suffix=": "/>
+        <group>
+          <text macro="author" suffix=", "/>
+          <choose>
+            <if type="thesis">
+              <group delimiter=", ">
+                <text macro="title" font-style="italic"/>
+                <text variable="genre"/>
+                <text variable="publisher"/>
+              </group>
+            </if>
+            <else-if type="article-journal article-magazine article-newspaper broadcast personal_communication thesis webpage" match="any">
+              <group delimiter=", ">
+                <text macro="title" quotes="true"/>
+                <group delimiter=" " prefix="in ">
+                  <text variable="container-title" form="short" font-style="italic"/>
+                  <group delimiter=", ">
+                    <text variable="volume"/>
+                    <text variable="issue"/>
+                  </group>
+                </group>
+                <text macro="pages"/>
+              </group>
+            </else-if>
+            <else-if type="book graphic" match="any">
+              <group delimiter=", ">
+                <text macro="title"/>
+                <group delimiter=" ">
+                  <text macro="publisher"/>
+                </group>
+              </group>
+            </else-if>
+            <else-if type="entry-dictionary entry-encyclopedia" match="any">
+              <group delimiter=", ">
+                <text macro="title" quotes="true"/>
+                <choose>
+                  <if variable="title-short" match="any">
+                    <text variable="title" form="short" font-style="italic" prefix="in "/>
```

Comment:
What's the expected output here? This seems odd

Reply:
It would be a version that would function until container-title-short is recognized for abridged titles.
For instance, the _Realencyclopädie der classischen Altertumswissenschaft_ has to be shortened in _RE_ but it cannot be done yet.
I guess I should had an explanation about this in the info section?

Reply:
Unless the extra field can be used to add a container-title-short field? But [there](https://www.zotero.org/support/kb/item_types_and_fields#citing_fields_from_extra), it doesn't seem possible (unless I can insert something like this:
`CSL Variable: Value`
`container-title-short: RE`)

Reply:
Yes, exactly: `container-title-short: RE` will work and is the right way to do this.

Reply:
Done. Travis seems to have difficulties to finish checking it though